lib/plugins/aws/provider/awsProvider.js

```diff
@@ -671,6 +671,18 @@ class AwsProvider {
             condition: { $ref: '#/definitions/awsResourceCondition' },
             dependsOn: { $ref: '#/definitions/awsResourceDependsOn' },
             description: { type: 'string', maxLength: 256 },
+            destinations: {
+              type: 'object',
+              properties: {
+                onSuccess: {
+                  anyOf: [{ type: 'string', minLength: 1 }, { $ref: '#/definitions/awsArnString' }],
```

Comment:
I think there's no point listing `awsArnString` where in all cases it matches `type: 'string'` (?)